index.js

```diff
@@ -46,6 +50,9 @@ var legacyModules = {
 };
 
 var configurations = {
+  'babel/register': {
+    extensions: '.js'
```

Comment:
can we get a note here explaining why we're capturing .js instead of babel.js (because of the `path.extname` thing you uncovered today)